hack/e2e-kind-cluster-setup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -xe
+
+CNI_VERSION=${CNI_VERSION:-0.4.0}
+OCI_BIN=${OCI_BIN:-docker}
+IMG_REGISTRY=${IMAGE_REGISTRY:-localhost:5000/maiqueb}
+
+setup_cluster() {
+    git clone https://github.com/k8snetworkplumbingwg/multus-cni/
+    trap "popd && cleanup" EXIT
```

Comment:
there is already trap for cleanup on line 30 isnt it ?

Reply:
I'll try to unify them.

Reply:
not sure you need, you just need to run the cleanup using the external trap, and remove from here, isn't it ?